op_map.cc

```diff
@@ -438,13 +438,14 @@ struct OpMapperBase : public vx::op_map::IOpMapper {
     if (broadcast_required) {
       int base_shape_idx = inputs[0]->GetShape().size() >
                   inputs[1]->GetShape().size()? 0 : 1;
-      std::vector<int32_t> broadcast_param;
+      std::vector<uint32_t> broadcast_param;
```

Comment:
rename  "broadcast" to reshape since we didn't do broadcast.

Reply:
done